src/DependencyInjection/Configuration.php

```diff
@@ -11,9 +11,6 @@ final class Configuration implements ConfigurationInterface
 {
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        $treeBuilder = new TreeBuilder('synolia_sylius_maintenance_plugin');
-        $rootNode = $treeBuilder->getRootNode();
-
-        return $treeBuilder;
+        return new TreeBuilder('synolia_sylius_maintenance_plugin');
```

Comment:
AFAIK, this Configuration class can be dropped if there is no config

Reply:
ookay